app/storage/src/test/java/com/fsck/k9/storage/messages/FolderHelpers.kt

```diff
@@ -15,7 +15,7 @@ fun SQLiteDatabase.createFolder(
     integrate: Boolean = false,
     inTopGroup: Boolean = false,
     displayClass: String = "NO_CLASS",
-    syncClass: String = "INHERITED",
+    syncClass: String? = "INHERITED",
```

Comment:
:+1: 

`notifyClass` and `pushClass` could be `null`, too. See https://github.com/k9mail/k-9/blob/783182a718c5a1d4c94e354f6e73941214addfbc/app/core/src/main/java/com/fsck/k9/mailstore/LocalStore.java#L898-L911

Reply:
Done